doc/code/expval/cv.rst

```diff
@@ -1,3 +1,4 @@
 .. automodule:: pennylane.expval.cv
    :members:
    :private-members:
+   :ignore-module-all:
```

Comment:
What is the reason behind adding this option, and removing :members:, :private-members:, and :inherited-members: in expval.rst?

Reply:
As far as I can tell, the built docs don't look different, so I assume this avoids some side-effect from the new organisation of the expval module?